### PR TITLE
[Typography] Added strong dependency to MDFTextAccessiblity for fallbackack logic when isLargeForContrastRatios: is unspecified.

### PR DESCRIPTION
**Pull request by randallli**

### CODE REVIEW

**Comment by randallli** [comment]
actually i think its better to just make it a non-optional method.
https://github.com/material-components/material-components-ios/pull/1251

File: components/Typography/src/MDCTypography.m
Comment (ianegordon) [inline components/Typography/src/MDCTypography.m]: nonnull UIFont to match the font loaders?

Otherwise, shortcircuit return NO if font == nil.

**Review by ianegordon — CHANGES_REQUESTED** [review]

File: components/Typography/src/MDCTypography.m
Comment (randallli) [reply components/Typography/src/MDCTypography.m]: fixed.
good catch.

**Review by ianegordon — APPROVED** [review]

**Comment by ianegordon** [comment]
Waiting on TravisCI before Merge.

**Pull request merged**
